### DISCUSSION TITLE
How to remove/modify link from sidebar logo in website?

### DISCUSSION THREAD

**Discussion by normansimonr** [quarto-dev/quarto-cli#2836] [Q&A] [answered]
I am creating a website and added a sidebar logo. When I click the logo, it takes me to the root folder. Is there a way I can tell Quarto not to add a link to the sidebar logo, or alternatively to redirect it to the `index.html` file?

Thanks!

**Comment by dragonstyle** [comment, marked as answer]
It should be routing to `index.hml`- thank you for catching this! I've made that change and it will be reflected in the next pre-release update (1.2.211 or later), which you can download here:

https://quarto.org/docs/download/prerelease.html

**Reply by normansimonr** [reply]
Thank you! It works!
